src/components/organisms/Header/__snapshots__/Header.spec.tsx.snap

```diff
@@ -6,6 +6,7 @@ exports[`renders correctly 1`] = `
     class="css-0"
   >
     <img
+      alt="images.bannerImage"
```

Comment:
Are you sure this works withou using the `t` function?

Reply:
Yes as this is the generated snapshot test. The test executes with no issues